src/libparsec/include/od_odt.h

```diff
@@ -9,351 +9,459 @@
 #ifndef _OD_ODT_H_
 #define _OD_ODT_H_
 
-
-
 //-----------------------------------------------------------------------------
-// ODT1 FORMAT (ODT)                                                          -
+// ODT1 FORMAT (ODT)							      -
 //-----------------------------------------------------------------------------
 
-
 // generic transformation matrix
-typedef fixed_t	ODT_Xmatrx[ 3 ][ 4 ];		// [0 0 0 1] (line 4) omitted!
-
+typedef fixed_t ODT_Xmatrx[3][4]; // [0 0 0 1] (line 4) omitted!
 
 // 3-D vertex coordinates
-struct ODT_Vertex3 {
+struct ODT_Vertex3
+{
 
-	fixed_t	X;
-	fixed_t	Y;
-	fixed_t	Z;
-	dword	Flags;	// padding to 16 bytes and flags
+	fixed_t X;
+	fixed_t Y;
+	fixed_t Z;
+	dword Flags; // padding to 16 bytes and flags
 };
 
-
 // 2-D point in unsigned integer coordinates (always >= 0)
-struct ODT_UPoint {
+struct ODT_UPoint
+{
 
-	dword	X;		// unsigned!
-	dword	Y;      // unsigned!
+	dword X; // unsigned!
+	dword Y; // unsigned!
 };
 
-
 // 2-D coordinates after projection; not yet converted to screen coordinates
-struct ODT_ProjPoint {
+struct ODT_ProjPoint
+{
 
-	fixed_t	X;
-	fixed_t	Y;		// 8 bytes
+	fixed_t X;
+	fixed_t Y; // 8 bytes
 };
 
-
 // 2-D point in screen coordinates (used in vertex list for the edge tracer)
-struct ODT_SPoint {
+struct ODT_SPoint
+{
 
-	long	X;		// signed!
-	long	Y;		// signed!
+	long X; // signed!
+	long Y; // signed!
 };
 
-
 // types for shading of faces
-enum ODT_shadingtype_t {
-
-	ODT_no_shad,			// lighting independent ambient color (no shading!)
-	ODT_flat_shad,			// flat shading
-	ODT_gouraud_shad,		// gouraud shading
-	ODT_afftex_shad,		// affine texture mapping
-	ODT_ipol1tex_shad,		// first order interpolated texture mapping (lin)
-	ODT_ipol2tex_shad,		// second order interpolated texture mapping (quad)
-	ODT_persptex_shad,		// perspective correct texmapping w/o interpolation
-	ODT_material_shad,		// shade using material specification
-	ODT_texmat_shad		// composite texture and material specification
-};
+enum ODT_shadingtype_t
+{
+
+	ODT_no_shad,			 // lighting independent ambient color (no shading!)
```

Comment:
The formatting and code style in a couple of these files got changed a bit but it's not a big deal

Reply:
I thought I had fixed it all, but perhaps some of it is still a bit messed up.  I think we can go through and fix it once we agree on some formatting/style stuff.  There may be clean up that is needed elsewhere anyway.

Reply:
It appears to be a difference of tabs vs expanded to space tabs.